### ISSUE TITLE
Horizontal line in editor

### ISSUE THREAD

**Issue by 5thHorseman** [Sigma88/Sigma-EditorView#6]
Just installed this on a fresh (well, GravityTurn's installed) KSP 1.8.1 install and there is an odd horizontal line on the floor. Above this line (on the screen) the floor is lighter than below it, as if it's being lit. It is horizontal on the SCREEN, and moves in a way that suggests it's position on the screen is based on the distance from the camera to the spot on the floor.

Other than that, wow that's a sexy VAB door.

![EditorView1](https://user-images.githubusercontent.com/5622822/78451154-bd628380-7651-11ea-8afa-3817a9eccb2c.jpg)

**Comment by 5thHorseman** [comment]
oops sorry forgot my log.
[KSP.log](https://github.com/Sigma88/Sigma-EditorView/files/4431343/KSP.log)

**Comment by Sigma88** [comment]
so it moves with your camera? when you turn the camera the line moves as well?

**Comment by Sigma88** [comment]
could you tell me the specific time it was in the space center when you entered the editor?

**Comment by 5thHorseman** [comment]
Yes it is always perfectly horizontal on the screen and moves with the camera. It shifts up and down vertically as I zoom in or out, or pan up or down.

I'll get the time and a couple more screenshots in a couple minutes.

**Comment by 5thHorseman** [comment]
It's Year 1 Day 3 2h 9m

If a picture is worth a thousand words, a video is probably worth 10,000. So here's a video.

https://www.youtube.com/watch?v=VYhW-cYSGcs

**Comment by Sigma88** [comment]
I can't reproduce your issue, are you using some setting or mod that could affect the editor scene?

**Comment by Sigma88** [comment]
could you try on a completely stock install + the latest release of this mod

**Comment by 5thHorseman** [comment]
I will. That was 0.4.0 but I did have one other mod installed, GravityTurn. This is my "test mods" install. I'll make sure I run it with GT and its two dependencies uninstalled.

**Comment by 5thHorseman** [comment]
Nope stock 1.8.1 with 0.4.0 as the only mod, same horizontal line.
[KSP.log](https://github.com/Sigma88/Sigma-EditorView/files/4431982/KSP.log)

**Comment by Sigma88** [comment]
can you take screenshots of your settings so that I can compare with my settings?

**Comment by 5thHorseman** [comment]
Here's the settings.cfg which I think has them all?

[settings.txt](https://github.com/Sigma88/Sigma-EditorView/files/4432572/settings.txt)

**Comment by 5thHorseman** [comment]
On a whim, i reset my settings to default and the line's not there. now to compare...

**Comment by 5thHorseman** [comment]
Comparing it "correct" with my original screenshot, it appears to be the "outside" lit up section at the door. Maybe the 1-2 meters of the VAB, where the door is, is more lit than the inside of the VAB.

I'm narrowing it down but it's a bit of a pain. I'm pretty sure it's one of these.
![image](https://user-images.githubusercontent.com/5622822/78462132-363afd00-769d-11ea-9a4c-a115c6ebc5cf.png)

**Comment by 5thHorseman** [comment]
Okay I have no idea why, but if you have
`QUALITY_PRESET = 2`
then it happens.

If instead you have
`QUALITY_PRESET = 5`
then it's fine.
All other graphics settings can be the same as on the right (my originals) but if the preset is not 5 it borks out.

I can't find a comparable setting in the game. This config is several versions old though so maybe it's a new default or something.

**Comment by 5thHorseman** [comment]
Found it. QUALITY_PRESET is "Render Quality" in the Graphics Settings. 5 is the max, "Fantastic". 2 is pretty low, "Fast".

It's obvious I don't care about graphics, I didn't even know I had set this low at some point :)

The line occurs when you have it set to 2 or less. 3, 4, and 5 look correct.

**Comment by Sigma88** [comment]
thanks a lot for narrowing it down

I will look into it, but honestly my understanding of how those things work is pretty low.

I can't promise you it will be possible to fix, in the meantime you should be able to keep it on 3 with not too much impact on your gameplay
